alerting/docker-compose.yml

```diff
@@ -25,6 +25,6 @@ services:
       - POSTGRES_USER=postgres
       - POSTGRES_DB=postgres
       - POSTGRES_PASSWORD=!ChangeMe!
-      - DATABASE_URL
-      - EMAIL_PASSWORD
-      - EMAIL_USER
+      - PRISM_ALERTS_DATABASE_URL
+      - ALERTING_EMAIL_PASSWORD=${ALERTING_EMAIL_PASSWORD} # Set from secrets in api/set_envs.sh
+      - ALERTING_EMAIL_USER=${ALERTING_EMAIL_USER}
```

Comment:
If these 3 are related, maybe rename them to use a common prefix to make this clearer? `PRISM_ALERTS_EMAIL_PASSWORD` and `PRISM_ALERTS_EMAIL_USER` ?

Reply:
Fixed ✅

Reply:
This does not appear to be fixed?